docs/pages/how_to_work_with_us.js

```diff
@@ -67,6 +67,37 @@ We ask that any ideas that change the functionality of an existing Gestalt compo
   </MainSection>,
 );
 
+card(
+  <MainSection name="How can you make design contributions to Gestalt?">
```

Comment:
Now that we have 3 pieces about contributing, maybe it'd help to organize them under a single MainSection called "Contributing", and then detail it with like subsections like  "Overall process", "Design changes", "Engineering changes" or something like that to create a little more organization?

Reply:
Actually maybe 4 subsections, adding one for "Other ways to contribute"

Reply:
OK, so I kept the `What is the process to request new additions or changes?` as a separate section as it's a little hard to unfurl the specifics of contributions in that section. 

The other three sections I popped into a `How can you contribute to Gestalt?` section. 

Let me know what you think!